mlir/include/air/Dialect/AIR/AIR.td

```diff
@@ -347,8 +347,8 @@ def air_DmaMemcpyNdOp: air_Op<"dma_memcpy_nd",
   let results = (outs Optional<air_AsyncToken>:$async_token);
   let assemblyFormat = [{
     custom<AsyncDependencies>(type($async_token), $async_dependencies)
-    `(` $dst `[` ($dst_offsets^)? `]``[` ($dst_sizes^)? `]``[` ($dst_strides^)? `]` `,`
-        $src `[` ($src_offsets^)? `]``[` ($src_sizes^)? `]``[` ($src_strides^)? `]` `)`  attr-dict `:`
+    `(` $dst (`<` $dst_offsets^ `>`)? (`<` $dst_sizes^ `>`)? (`<` $dst_strides^ `>`)? `,`
```

Comment:
I'm not sold on the angle brackets. What are the arguments for the change?

Reply:
The use of square brackets may look like array indexing. Angle brackets could clear this confusion.